admin/controllers/school.js

```diff
@@ -20,6 +21,11 @@ const controller = {}
 controller.getSchoolLandingPage = async (req, res, next) => {
   res.locals.pageTitle = 'School Homepage'
   try {
+    const hasHelpdeskNotReceivedImpersonation = helpdeskService.hasHelpdeskNotReceivedImpersonation(req.user)
```

Comment:
I think naming things in the negative is generally harder to understand.  Would be better as `!isHelpdeskImpersonator()` IMHO.  Happy to discuss.

Reply:
@jon-shipley this is now simplified and split to two methods.